src/main/java/com/simibubi/create/foundation/tileEntity/behaviour/filtering/FilteringHandler.java

```diff
@@ -95,7 +95,7 @@ else if (ItemHandlerHelper.canItemStacksStack(toApply, filter))
 			Component formattedText = new TranslatableComponent(translationKey);
 			player.displayClientMessage(Lang.createTranslationTextComponent("logistics.filter." + feedback, formattedText)
 				.withStyle(ChatFormatting.WHITE), true);
-		}
+		//}
```

Comment:
this screams broken - see above.

Reply:
this allows the filter feedback text to be sent no matter if its a integrated server or dedicated server